README.md

```diff
@@ -18,6 +18,24 @@ See our full documentation: [http://hermes-pubsub.readthedocs.org/en/latest/](ht
 
 Questions? We are on [gitter](https://gitter.im/allegro/hermes).
 
+## Development
+
+File docker/docker-compose.development.yml disables deployment of hermes frontend, management, and consumers.
```

Comment:
The problem with putting this section in README is that there is no additional context here, e.g. what frontend, management, and consumers are. 

In my opinion, we should extend [the Quickstart section from the documentation](https://hermes-pubsub.readthedocs.io/en/latest/quickstart/#quickstart).